.circleci/images/primary/Dockerfile-2.5.9

```diff
@@ -1,12 +1,6 @@
 # Note: See the "Publishing updates to images" note in ./README.md for how to publish new builds of this container image
 
-FROM ruby:2.5.9-buster
-
-# Make apt non-interactive
-RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90circleci \
-  && echo 'DPkg::Options "--force-confnew";' >> /etc/apt/apt.conf.d/90circleci
-
-ENV DEBIAN_FRONTEND=noninteractive
+FROM ghcr.io/datadog/images-rb/engines/ruby:2.5
```

Comment:
I'm curious: where's the source for these `ghcr.io/datadog/images-rb/engines/ruby` images?

Reply:
Here: https://github.com/DataDog/images-rb